SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialNetDriver.cpp

```diff
@@ -508,6 +510,9 @@ void USpatialNetDriver::CreateAndInitializeCoreClasses()
 	check(NewPackageMap == PackageMap);
 
 	PackageMap->Init(this, &TimerManager);
+
+	//Pass the PackageMap to the InterestFactory. The PackageMap was not available to pass when the InterestFactory was created.
```

Comment:
I arrange it this way just to coincide with when things are initialized.
CreateClientCheckoutRadiusConstraint is called on the interest factory on USpatialNetDriver::InitBase before we have a spatial connection I believe. I wanted to avoid making CreateClientCheckoutRadiusConstraint static and initialising the InterestFactory later so I initialed it her. 

The package map is taken from the SpatialOSConnection. This is done much later and is why I chose to parse it to the InterestFactory. Not sure what we pref here.